controllers/chat/ChatController.js

```diff
@@ -426,6 +426,135 @@ module.exports = {
       return ErrorResponse.e400(res, error.message);
     }
   },
+  initChatMoveToAnon: async (req, res) => {
+    const {
+      targetUser,
+      message,
+      anon_user_info_color_code,
+      anon_user_info_color_name,
+      anon_user_info_emoji_code,
+      anon_user_info_emoji_name
+    } = req.body;
+
+    let members = [targetUser];
+    if (!members.includes(req.userId)) members.push(req.userId);
+
+    const client = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
```

Comment:
mas @brm-stnd please move this function to own file so it doesn't make chatController grow too big